schemas/Semantic.schema.json

```diff
@@ -1232,6 +1232,14 @@
                     "description": "The place has been given a private label by the user. See: [Give a place a private label](https://support.google.com/maps/answer/6257830?hl=en).",
                     "extra_color": "#03a9f4",
                     "extra_icon": "semantic_types/label_circled_24dp.svg"
+                },
+                {
+                    "type": "string",
+                    "const": "TYPE_UNKNOWN",
+                    "title": "Type Unknown Location",
+                    "description": "Location type is unknown",
+                    "extra_color": "#03a9f4",
+                    "extra_icon": "semantic_types/unknown_24dp.svg"
```

Comment:
Same here - the icon does not exists in https://github.com/CarlosBergillos/LocationHistoryFormat/tree/main/docs/static/icons/semantic_types - what should I use here?

Reply:
Oh, I haven't encountered this one!
In this case (semantic type) the icons are also based on what is shown in the web interface.

![Screenshot_20240401_194925](https://github.com/CarlosBergillos/LocationHistoryFormat/assets/1731534/9d483d69-2516-4635-bdb5-ba66f4505ea2)

If you have access to the Google account where this `TYPE_UNKNOWN` was found, can you navigate to that day in the Google Timeline web interface and see how it looks there?

Reply:
It looks very similar.

![Screenshot 2024-04-13 232530](https://github.com/CarlosBergillos/LocationHistoryFormat/assets/99822/e77c7fcf-0382-4f44-af59-ebcd73d2421d)

This type shows up in the `"otherCandidateLocations"` location items. Eg.

```json
{
    "placeVisit": {
        "location": {
            "semanticType": "TYPE_HOME",
            "locationConfidence": 100.0,
            "calibratedProbability": 100.0
        },
        "duration": {
            "startTimestamp": "2024-03-04T02:34:29Z",
            "endTimestamp": "2024-03-04T13:42:14Z"
        },
        "placeConfidence": "USER_CONFIRMED",
        "visitConfidence": 100,
        "otherCandidateLocations": [
            {
                "placeId": "Ch...wY",
                "address": "Address, City, State 12345, Country",
                "name": "Place Name",
                "semanticType": "TYPE_UNKNOWN",
                "locationConfidence": 37.953373,
                "calibratedProbability": 37.953373
            }
        ],
        "editConfirmationStatus": "CONFIRMED",
        "locationConfidence": 100,
        "placeVisitType": "SINGLE_PLACE",
        "locationAssertionType": "WITHIN_OR_AT",
        "lastEditedTimestamp": "2024-03-13T00:37:59.560Z",
        "placeVisitImportance": "MAIN"
    }
}
```